### PR TITLE
ENCD-4435 Add ENCv5 ccre internal_tags to datasets

### PR DESCRIPTION
**Pull request by zmyers23**
Added 'ENCYCLOPEDIAv5' and renamed 'cre' enums for internal_tags property of dataset objects.

### CODE REVIEW

File: src/encoded/tests/test_upgrade_dataset.py [hunk 1]
```diff
@@ -243,6 +243,7 @@ def experiment_22(root, experiment):
         'biosample_type': 'primary cell',
         'biosample_term_id': 'CL:0000765',
         'biosample_term_name': 'erythroblast',
+        'internal_tags': ['cre_inputv10','cre_inputv11', 'ENCYCLOPEDIAv3'],
```
[the remark this answers is not in the document]
Comment (zmyers23) [reply src/encoded/tests/test_upgrade_dataset.py]: removed space.

File: src/encoded/tests/test_upgrade_dataset.py [hunk 2]
```diff
@@ -257,6 +258,7 @@ def annotation_20(award, lab):
         'biosample_type': 'primary cell',
         'biosample_term_id': 'CL:0000765',
         'biosample_term_name': 'erythroblast',
+        'internal_tags': ['cre_inputv10','cre_inputv11', 'ENCYCLOPEDIAv3']
```
[the remark this answers is not in the document]
Comment (zmyers23) [reply src/encoded/tests/test_upgrade_dataset.py]: removed space.

**Comment by zmyers23** [comment]
> Some linting issues. Do you have a linter running in your text editor? We should set one up later. You'll want to make sure there are no lint issues on lines that you edit.
> 
> Shorten up the PR name, keep them on one line.

I don't think I have a linter but it sounds helpful.

**Comment by yunhailuo** [comment]
I think @caseylitton meant to be the opposite by saying "space after comma.", i.e. you should have a space after comma instead of removing it, right?

**Comment by zmyers23** [comment]
> I think @caseylitton meant to be the opposite by saying "space after comma.", i.e. you should have a space after comma instead of removing it, right?

In looking through the document, space after comma is the correct format. I have added a space after each comma.

File: src/encoded/tests/test_upgrade_dataset.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (zmyers23) [reply src/encoded/tests/test_upgrade_dataset.py]: added space.

File: src/encoded/tests/test_upgrade_dataset.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (zmyers23) [reply src/encoded/tests/test_upgrade_dataset.py]: added space.

File: src/encoded/tests/test_upgrade_dataset.py [hunk 3]
```diff
@@ -533,3 +535,17 @@ def test_upgrade_annotation_20_to_21(root, upgrader, annotation_20, erythroblast
...
+        'experiment', experiment_22, current_version='23', target_version='24'
+    )
+    assert value['internal_tags'] == ['ccre_inputv1','ccre_inputv2', 'ENCYCLOPEDIAv3']
+
+
+def test_upgrade_annotation_21_to_22(root, upgrader, annotation_20):
+    value = upgrader.upgrade(
+        'annotation', annotation_20, current_version='21', target_version='22'
+    )
+    assert value['internal_tags'] == ['ccre_inputv1','ccre_inputv2', 'ENCYCLOPEDIAv3']
```
[the remark this answers is not in the document]
Comment (zmyers23) [reply src/encoded/tests/test_upgrade_dataset.py]: added space.

**Pull request merged**